apps/paf/translations/src/en/validation.json

```diff
@@ -271,6 +271,12 @@
   "how-do-you-know-the-person": {
     "maxlength": "You can't use more than {{maxlength}} characters for your answer"
   },
+  "about-you-dob": {
+    "date": "Please enter a valid date",
+    "required": "Enter your date of birth",
```

Comment:
Remove the required message as the date of birth is not mandatory.

Reply:
"required' message removed.